README.md

```diff
@@ -161,6 +161,30 @@ func ExampleClient() *redis.Client {
 
 ```
 
+
+### Advanced Configuration
+
+go-redis supports extending the CLIENT SETNAME command to allow for custom client identification. This feature enables community members to append their own suffixes to the client name, facilitating a deeper understanding of Redis usage patterns.
+
+#### Default Client Identification
+
+By default, go-redis automatically sets the client library name and version during the connection process. This behavior ensures that the Redis server is aware of the client's identity, which is crucial for monitoring, debugging, and analytics.
+
+#### Disabling Identity Verification
+
+For use cases where connection identity verification is not required or needs to be explicitly disabled, go-redis provides the `DisableIdentity` configuration option. This option can be particularly useful in environments where Redis servers do not support `client setinfo`.
```

Comment:
When connection identity verification is not required or needs to be explicitly disabled, a `DisableIdentity` configuration option exists.